keyboards/hardlight/keymaps/default/keymap.c

```diff
@@ -0,0 +1,132 @@
+
+#include QMK_KEYBOARD_H
+
+#define _MAIN 0
+#define _RAISE 1
+#define _LOWER 2
+#define _DVORAK 3
+#define _SET 4
```

Comment:
```suggestion
enum layers {
    _MAIN,
    _DVORAK,
    _RAISE,
    _LOWER,
    _SET,
};
```

Prefer an `enum` here for ease of layer management.

Also, the way these are currently arranged, `_DVORAK` layer will not be able to reach `_RAISE` or `_LOWER` by means of `RAISE` and `LOWER` defines (you can only move "up" layers with `MO`).  I have instead suggestd placing `_DVORAK` above `_QWERTY` but below the other layers.